asset_manager/fetch.py

```diff
@@ -146,28 +140,30 @@ def dollars_to_float(dollar_str):
 liability_df = convert_dollar_cols_to_float(liability_df)
 
 # Merge into a single DF so we can group and sum.
-asset_df['Type'] = 'asset'
-liability_df['Type'] = 'liability'
-liability_df['Accessible'] = 'Y'
+asset_df["Type"] = "asset"
+liability_df["Type"] = "liability"
+liability_df["Accessible"] = "Y"
 full_df = pd.concat([asset_df, liability_df])
 # Sum accessible assets and liabilities.
-grouped: GroupedType = full_df[full_df['Accessible'] == 'Y'].groupby('Type')
-accessible = grouped[['Amount', 'Precision (+/-)']].sum()
+grouped: GroupedType = full_df[full_df["Accessible"] == "Y"].groupby("Type")
+accessible = grouped[["Amount", "Precision (+/-)"]].sum()
 accessible_amt = (
-    accessible.loc['asset', 'Amount'] - accessible.loc['liability', 'Amount']
+    accessible.loc["asset", "Amount"] - accessible.loc["liability", "Amount"]
 )
-accessible_precision = accessible['Precision (+/-)'].sum()
+accessible_precision = accessible["Precision (+/-)"].sum()
 
 # Sum *all* assets and liabilities.
-grouped = full_df.groupby('Type')
-equity = grouped[['Amount', 'Precision (+/-)']].sum()
-equity_amt = equity.loc['asset', 'Amount'] - equity.loc['liability', 'Amount']
-equity_precision = equity['Precision (+/-)'].sum()
-
-todays_date = datetime.date.today().isoformat().replace('-', '_')
-object_name = f'summaries_{todays_date}.csv'
+grouped = full_df.groupby("Type")
+equity = grouped[["Amount", "Precision (+/-)"]].sum()
+equity_amt = equity.loc["asset", "Amount"] - equity.loc["liability", "Amount"]
+equity_precision = equity["Precision (+/-)"].sum()
+
+todays_date = datetime.date.today().isoformat().replace("-", "_")
+object_name = f"summaries_{todays_date}.csv"
+print("Writing DataFrame...")
+print(full_df.reset_index(drop=True).to_string())
```

Comment:
Now the dataframe is printed to the console before being stored in S3. This code is awful but I'm not about to rewrite the whole project right now.